target/rspamd/local.d/settings.conf

```diff
@@ -0,0 +1,12 @@
+# documentation: https://rspamd.com/doc/configuration/settings.html
+
+# DMS::SED_TAG::1::START
+# Disable all checks for authenticated users
+authenticated {
+    priority = high;
+    authenticated = yes;
+    apply {
+        groups_enabled = [];
```

Comment:
This looks okay to me, I will try to test it in production. No promises but I will do my best 

Not sure if some kind of forwarding outbound flow could be affected

Reply:
Nice. I'll merge when you sent me feedback about this :)